src/app/products/fixed-deposit-products/view-fixed-deposit-product/view-fixed-deposit-product.component.html

```diff
@@ -0,0 +1,421 @@
+        </div>
+
+        <div fxFlexFill>
+          <span fxFlex="40%">Days in Years:</span>
+          <span fxFlex="60%">{{ fixedDepositProductData.interestCalculationDaysInYearType.value }}</span>
+        </div>
+
+        <hr>
+
+        <h3 class="mat-h3" fxFlexFill>Settings</h3>
+
+        <mat-divider [inset]="true"></mat-divider>
+
+        <div fxFlexFill>
+          <span fxFlex="40%">Lock-in Period:</span>
+          <span fxFlex="60%" *ngIf="fixedDepositProductData.lockinPeriodFrequency !== undefined">
+            {{ fixedDepositProductData.lockinPeriodFrequency }}
+            &nbsp;
+            {{ fixedDepositProductData.lockinPeriodFrequencyType.value }}</span>
+        </div>
+
+        <div fxFlexFill>
+          <span fxFlex="40%">Minimum Deposit Term:</span>
+          <span fxFlex="60%"
+            *ngIf="fixedDepositProductData.minDepositTerm !== undefined">{{ fixedDepositProductData.minDepositTerm }}
+            &nbsp;
+            {{ fixedDepositProductData.minDepositTermType.value }}</span>
+        </div>
+
+        <div fxFlexFill>
+          <span fxFlex="40%">Maximum Deposit Term:</span>
+          <span fxFlex="60%"
+            *ngIf="fixedDepositProductData.maxDepositTerm !== undefined">{{ fixedDepositProductData.maxDepositTerm }}
+            &nbsp;
+            {{ fixedDepositProductData.maxDepositTermType.value }}</span>
+        </div>
+
+        <div fxFlexFill>
+          <span fxFlex="40%">Deposit Term In multiple of:</span>
+          <span fxFlex="60%"
+            *ngIf="fixedDepositProductData.inMultiplesOfDepositTerm !== undefined">
+            {{ fixedDepositProductData.inMultiplesOfDepositTerm }}
+            -
+            {{ fixedDepositProductData.inMultiplesOfDepositTermType.value }}</span>
+        </div> 
+
+        <div fxFlexFill *ngIf="fixedDepositProductData.preClosurePenalInterest">
+          <span fxFlex="40%">Apply penal interest (less):</span>
+          <span fxFlex="60%">{{fixedDepositProductData.preClosurePenalInterest}} &nbsp; &nbsp; % &nbsp; &nbsp; On
+            &nbsp;&nbsp;
+            {{fixedDepositProductData.preClosurePenalInterestOnType.value}}</span>
+        </div>
+
+        <div fxFlexFill *ngIf="fixedDepositProductData.minBalanceForInterestCalculation">
+          <span fxFlex="40%">Balance Required For Interest Calculation:</span>
+          <span fxFlex="60%">{{ fixedDepositProductData.minBalanceForInterestCalculation | number }}</span>
+        </div>
+
+        <div fxFlexFill>
+          <span fxFlex="40%">Is withhold tax Applicable:</span>
+          <span fxFlex="60%">{{ fixedDepositProductData.withHoldTax }}</span>
+        </div>
+
+        <div fxFlexFill *ngIf="fixedDepositProductData.withHoldTax">
+          <span fxFlex="40%">Withhold Tax group:</span>
+          <span fxFlex="60%">{{ fixedDepositProductData.taxGroup.name }}</span>
+        </div>
+
+        <hr>
+
+        <h3 class="mat-h3" fxFlexFill>Interest Rate Chart</h3>
```

Comment:
Have a look at the current implementation of Interest Rate Charts in the web app, in Preview of Create Fixed Deposit Product and update this accordingly. All interest rate chart slabs and incentives should be visible in table view here.